ARM/Ch.12/radon_group_chr.stan

```diff
@@ -1,17 +1,17 @@
 data {
-  int<lower=0> N; 
+  int<lower=0> N;
   int<lower=1,upper=85> county[N];
```

Comment:
@ericnovik not a big deal, but might as well use `J` here too instead of `85` like you do in the others